python/ray/tune/util.py

```diff
@@ -97,6 +98,43 @@ def stop(self):
         self.stopped = True
 
 
+class PriorityQueue(object):
```

Comment:
is this just a helper class around heapq?

Reply:
yeah, similar to python's `queue.PriorityQueue` without the unnecessary locking

Reply:
i can move all of it into `Trainable`, just a bit cleaner this way